var Exception = function Exception(code, err) {
    if (err) console.log(err);

    console.log('Error', code, ':', I.ExceptionCodes[code]);
    Error.call(this, code);
    throw this;
};

Exception.prototype = new Error();
Exception.prototype.constructor = Exception;

exports.Exception = Exception;
